patches/server/0928-More-vanilla-friendly-methods-to-update-trades.patch

```diff
@@ -0,0 +1,124 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Lulu13022002 <41980282+Lulu13022002@users.noreply.github.com>
+Date: Sun, 16 Oct 2022 16:12:49 +0200
+Subject: [PATCH] More vanilla friendly methods to update trades
+
+
+diff --git a/src/main/java/net/minecraft/world/entity/npc/AbstractVillager.java b/src/main/java/net/minecraft/world/entity/npc/AbstractVillager.java
+index 5eab7d50734551d96128dfebee126a1da4c51375..562bbd9d1b7595bf5d5c175c42cc5ff3f81ee9d9 100644
+--- a/src/main/java/net/minecraft/world/entity/npc/AbstractVillager.java
++++ b/src/main/java/net/minecraft/world/entity/npc/AbstractVillager.java
+@@ -257,8 +257,13 @@ public abstract class AbstractVillager extends AgeableMob implements InventoryCa
+ 
+         return j >= 0 && j < this.inventory.getContainerSize() ? SlotAccess.forContainer(this.inventory, j) : super.getSlot(mappedIndex);
+     }
++    // Paper start
++    protected void updateTrades() {
++        updateTrades(2);
++    }
+ 
+-    protected abstract void updateTrades();
++    public abstract void updateTrades(int amount);
```

Comment:
This should return a boolean on whether trades were actually added (for use in the boolean API return value)